samtranslator/model/api/http_api_generator.py

```diff
@@ -24,8 +24,8 @@
 )
 CorsProperties.__new__.__defaults__ = (None, None, None, None, None, False)
 
-AuthProperties = namedtuple("_AuthProperties", ["Authorizers", "DefaultAuthorizer"])
-AuthProperties.__new__.__defaults__ = (None, None)
+AuthProperties = namedtuple("_AuthProperties", ["Authorizers", "DefaultAuthorizer", "EnableIamAuthorizer"])
+AuthProperties.__new__.__defaults__ = (None, None, None)
```

Comment:
Looking at this now: should the last `None` here actually be `False`?

Reply:
`None` is falsy so they both work the same. It would only matter if `None` had some value to be considered different. For this, I don't think it strictly matters.

Reply:
Would it be better to use False instead of None largely to make it clear we expect a boolean value?